integrated_channels/canvas/docs/decisions/0001-integration-classic-vs-lti.rst

```diff
@@ -0,0 +1,37 @@
+Canvas Integration via Classic Integration model vs LTI
```

Comment:
Might be worth expanding on what LTI is here, have seen good form to be first usage of an acronym to spell out what it means next to it or set up part of the doc where you have definitions.